src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
В тебе завжди **health = 100**, тому можна не передавати його в аргументи
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
```

Reply:
согласна. здоровье всегда 100 по дефолту, оно не передается параметром